src/components/CalendarDay.js

```diff
@@ -1,26 +1,28 @@
 import React from 'react';
 import classnames from 'classnames';
 
-const CalendarDay = ({day, isToday, events, onClick}) => {
+class CalendarDay extends React.Component {
+  render () {
     const dayClasses = classnames({
         'flexColumn': true,
         'day': true,
-        'inactive': day.siblingMonth,
-        'today': isToday,
+        'inactive': this.props.day.siblingMonth,
+        'today': this.props.isToday,
```

Comment:
Actually it's different across the files. In `CalendarEvent` it's 2, in `index.js` it's 4.